src/networks/starknet/queries.ts

```diff
@@ -170,8 +170,8 @@ export const SPACE_QUERY = gql`
 `;
 
 export const SPACES_QUERY = gql`
-  query {
-    spaces {
+  query ($first: Int!, $skip: Int!) {
+    spaces(first: $first, skip: $skip, orderBy: created, orderDirection: desc) {
```

Comment:
Looks good! Can you just order by vote_count here?
```suggestion
    spaces(first: $first, skip: $skip, orderBy: vote_count, orderDirection: desc) {
```